pkg/caretta/links_tracer.go

```diff
@@ -31,6 +31,14 @@ var (
 		Name: "caretta_current_loopback_connections",
 		Help: `Number of loopback connections observed in the last iteration`,
 	})
+	mapSize = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "caretta_ebpf_map_size",
```

Comment:
I'd rename this to "caretta_ebpf_connection_map_size", to distinguish from the sock info map.

Reply:
Also - I don't remember if it's possible to get a map size from user mode, but if it's possible - I'd use it instead of the counter, plus update it for both maps (and not just the one the user mode iterates).

Reply:
renamed